cConsole/.vscode/launch.json

```diff
@@ -38,6 +38,9 @@
             "environment": [],
             "externalConsole": false,
             "internalConsoleOptions": "openOnSessionStart",
+            "logging": {
+                "engineLogging": true,  // set this to false for a smaller log on the debug console
```

Comment:
Sounds good, I will remove it

Reply:
Actually I am looking at the JSON files which have comments, they are pretty much all related to VSCode (almost all are at `.vscode/launch.json`), and some of the comments are even added by VSCode, like this task on `aspnetBlazor` for example:
```
        {
            // Use IntelliSense to find out which attributes exist for C# debugging
            // Use hover for the description of the existing attributes
            // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
            "name": "Local Debug",
            "type": "coreclr",
            "request": "launch",
            "preLaunchTask": "build-debug-local",
            // If you have changed target frameworks, make sure to update the program path.
            "program": "${workspaceFolder}/bin/Debug/net8.0/__change__.dll",
            "args": [],
            "cwd": "${workspaceFolder}",
            "stopAtEntry": false,
            // Enable launching a web browser when ASP.NET Core starts. For more information: https://aka.ms/VSCode-CS-LaunchJson-WebBrowser
            "serverReadyAction": {
                "action": "openExternally",
                "pattern": "\\bNow listening on:\\s+(https?://\\S+)"
            },
            "env": {
                "ASPNETCORE_ENVIRONMENT": "Development"
            },
            "sourceFileMap": {
                "/Views": "${workspaceFolder}/Views"
            }
        },
```
So in this files that are just related to VSCode I think we can keep the comments, as they can be helpful for the user.

Reply:
you are right, as I said this sounds informative. But we have also to think when these are used on scripts and outside VS Code. So, because of this, I'm more oriented to follow what all the parsers out there follow.

Reply:
But is `launch.json` used anywhere outside VSCode? I tend to like the comments on this one if it's used just within VSCode